examples/shader/shader_material.rs

```diff
@@ -105,10 +102,7 @@ fn setup(
             transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
             ..default()
         },
-        DepthPrepassSettings {
-            output_normals: true,
-            depth_resource: true,
-        },
+        PrepassSettings::default(),
```

Comment:
🤔 I wonder if that should just be part of the Camera3dBundle

Reply:
You don't need to do this now, just an open question

Reply:
Maybe as an Option<PrepassSettings>? I think the idea is that you only added it if needed, so that if you aren't using SSAO/SSR/etc, and don't care about overdraw, then you don't need to do the prepass. Not sure how excluding it is different vs PrepassSettings with all fields set to false. That's another thing I'm confused on.